server/cmwell-spa/utils/upload.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+tokenFileName=~/dev/uploadTool/token-$1
+[ -e "$tokenFileName" ] || touch $tokenFileName -d 2000-01-01
+
+tokenAge=$(($(date +%s) - $(date +%s -r $tokenFileName)))
+if [ $tokenAge -gt 3600 ]
+then
+    token=$(curl -s "http://${1:-localhost:9000}/_login" -u root | jq -r ".token")
+    echo $token > $tokenFileName
+else
+    token=$(cat $tokenFileName)
+fi
+
+# Create AppCache Manifest according to file list and commit hash
+
+commit=$(git rev-parse HEAD)
+fileList=$(find -type f | grep -vE 'index.html|cmwell.appcache' | sed 's/^\./\/meta\/app\/main/')
+echo -e "CACHE MANIFEST\n\n# $commit\n\nCACHE:\n$fileList\n\nNETWORK:\n*" > cmwell.appcache
+
+# Upload SPA
+
+for file in $( find -newermt `date +%Y-%m-%d` -type f | grep -v ".idea\|target" | sed 's/^.\///' ); do
```

Comment:
Lets filter all hidden dirs, not just `.idea`

Reply:
I added some documentation - each one can ignore what ever they like to.